tools/xray_flux_forecast/v4_find_ys.py

```diff
@@ -33,22 +40,30 @@
 Y_data = list(Y_data)
 Y_init_date = datetime.datetime.strptime(Y_data[1][0],'%Y-%m-%d %H:%M:%S')
 
+
+###############################
+#                             #
+#  Delay and Catch definition #
+#                             #
+###############################
+
 delays = [60]
-durations = [12, 36, 60, 24*60]
-#durations = [12, 36, 60, 24*60]
+catches = [12, 36, 60, 24*60]
+#catches = [12, 36, 60, 24*60]
 
 for delay in delays:
-    for duration in durations:
+    for catch in catches:
         
         def get_Y_index(date):
+            """Function that finds the index on the large y file for a given date"""
```

Comment:
Good function comments have a brief description, followed by the type definition and description of the parameters. I cannot read this comment and understand the purpose of the function, what code will likely call the function, and how I should call the function if I want to adapt it to another use in the future.